src/components/presentation/RatingArticles.test.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { fireEvent, render } from 'react-testing-library';
+import { Provider } from 'react-redux';
+import 'jest-dom/extend-expect';
```

Comment:
Hi @marcdomain, Nice implementation of your feature. Consider removing the import on line 4. There is a setupTest file already handling for that.

Reply:
Thanks for the heads up. It has been updated